### ISSUE TITLE
Address JH accessibility issues

### ISSUE THREAD

**Issue by samuelscandrett** [k-int/arches-jersey#36]
Firstly, we need to ensure accessibility mode is set to True within the project settings.py. We use settings_local to overwrite with project secrets, but this is a generic setting that needs to be switched on.
https://github.com/k-int/arches-jersey/blob/661592c54272cf1d53d26b3e7a455cdba7efe3a1/jersey_her/settings.py#L300

The index.htm has 5 major errors, 1 contrast error and 7 alerts, as outlined by Wave:

<img width="1892" height="887" alt="Image" src="https://github.com/user-attachments/assets/86b81fd7-6c5b-4d86-8774-0218321cb0ad" />

There are also 5 failures for contrast checks when using WCAG Contrast Checker

<img width="1895" height="907" alt="Image" src="https://github.com/user-attachments/assets/104347eb-1b06-48df-9b84-029b8419e30d" />

On the Arches search page (and others), we have 2 errors caused by the banner that extends base-manager. I believe these are due to a missing alt label on the Jersey Heritage logo.

<img width="857" height="652" alt="Image" src="https://github.com/user-attachments/assets/6a0c6d6d-d35d-4193-a92b-694ee4f7b3e0" />
 

Tools used are [Wave](https://wave.webaim.org/) and [Contrast checker](https://webaim.org/resources/contrastchecker/), both Chrome extensions. A full list of brower extensions can be found on the Arches documentation: https://arches.readthedocs.io/en/stable/developing/advanced/accessibility/#tools-used.
